port/cplstringlist.cpp

```diff
@@ -107,6 +156,27 @@ CPLStringList::CPLStringList(CPLStringList &&oOther) : CPLStringList()
     operator=(std::move(oOther));
 }
 
+/************************************************************************/
+/*                           BoundToConstList()                         */
+/************************************************************************/
+
+/**
+ * Return a CPLStringList that wraps the passed list.
+ *
+ * The input list is *NOT* copied and must be kept alive while the
+ * return CPLStringList is used.
+ *
+ * @param papszListIn a NULL terminated list of strings to wrap into the CPLStringList
```

Comment:
```suggestion
 * @param papszListIn a list of NULL terminated strings to wrap into the CPLStringList
```

Reply:
the original phrasing is actually correct. The list is terminated by a NULL pointer. This is a phrasing we use in a lot of places. Each string is assumed to be terminated by the NUL character, but that's kind of obvious to any user of a C API.